### PR TITLE
feat: Add opt-in root exit spans to logging

### PR DESCRIPTION
**Pull request by Ferenc-**

### CODE REVIEW

File: src/instana/instrumentation/logging.py [hunk 1]
```diff
@@ -18,32 +19,34 @@ def log_with_instana(wrapped, instance, argv, kwargs):
...
-
-            # create logging span
-            with active_tracer.start_active_span('log', child_of=active_tracer.active_span) as scope:
-                scope.span.log_kv({ 'message': msg })
-                if parameters is not None:
-                    scope.span.log_kv({ 'parameters': parameters })
-                # extra tags for an error
-                if argv[0] >= logging.ERROR:
-                    scope.span.mark_as_errored()
+        if tracing_is_off() or argv[0] < logging.WARN:
```
[the remark this answers is not in the document]
Comment (Ferenc-) [reply src/instana/instrumentation/logging.py:25]: Yes [conforming the current spec](https://github.ibm.com/instana/technical-documentation/tree/master/tracing/specification#log-spans) `By default (in the absence of any configuration, see below), tracers MUST only capture log spans with severity warn or higher. `.

File: src/instana/instrumentation/logging.py [hunk 2]
```diff
@@ -18,32 +19,34 @@ def log_with_instana(wrapped, instance, argv, kwargs):
     # argv[2] = args for message
     try:
         active_tracer = get_active_tracer()
```
[the remark this answers is not in the document]
Comment (Ferenc-) [reply src/instana/instrumentation/logging.py:21]: Good catch! This was a git-fu failure. Fixed.

File: src/instana/instrumentation/logging.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (pvital) [reply src/instana/instrumentation/logging.py:23]: Perfect. After reading the specification more thoroughly, it's clear that even when configuring the log level, only `WARNING`, `ERROR`, and `CRITICAL` are valid entries.

> Tracers MAY offer to configure the minimum log level that is captured. When an environment variable is offered to configure this, its name MUST be `INSTANA_LOG_LEVEL_CAPTURE`.
>
> Valid configuration values for this MUST be restricted to the following list:
>
> - `WARN` (which is the default),
> - `ERROR` (only capture log spans for severity error or higher), and
> - `OFF` (do not capture log spans at all).
>
> Matching on the values MUST be done in a case-insensitive fashion. If the provided value is not valid, the tracer SHOULD log a warning and say which values are valid. (Example: `The value 'INFO' for 'INSTANA_LOG_LEVEL_CAPTURE' is not valid, valid values are 'WARN', 'ERROR', and 'OFF'.`)
>
> That is, customers can only use this configuration to capture less log spans, not more -- they cannot configure to also capture info or debug logs as spans.

**Review by pvital — APPROVED** [review]
It looks good to me.

**Pull request merged**